mongo-initiator/5.0/Dockerfile

```diff
@@ -6,5 +6,6 @@ COPY ./docker-entrypoint.sh /
 
 USER mongodb
 
+RUN chmod +x /docker-entrypoint.sh
```

Comment:
hey @ani-sha @gunnarmorling  do we actually still need an initiator image? I think we can solve what we do here with a startup script for the Mongo DB image and use a standard mongo image to initialize replication. In Debezium UI we do it like that:
https://github.com/debezium/debezium-ui/blob/master/docker-compose.yml#L48-L61

Reply:
Oh, yes, would love to get rid of this, if we can. But let's handle this separately; could you log a Jira issue please, linking to your solution above. Thx!

Reply:
I created that issue in December^^

Reply:
Re the fix, can't we make the actual file itself executable, instead of doing this only in the Dockerfile, @ani-sha?

Reply:
@gunnarmorling Exactly, it is the same for other versions

Reply:
> I created that issue in December^^

Can you share the link please?

Reply:
It's mentioned in my first reply.

Reply:
> can't we make the actual file itself executable

@gunnarmorling the `docker-entrypoint.sh` file?

Reply:
Yes, exactly.

Reply:
Ah, thx, @rk3rn3r, missed that :)

Reply:
I am not aware of how to do it. Will `set -eEo+x pipefail` achieve the same?

Reply:
You'd just run the same `chmod` command on your machine (assuming you're on Linux or macOS); Git should pick up the changed permission flag and you can commit that change.

The `set ...` stuff is unrelated to that; it's for executing shell scripts with safer settings, like aborting early after a failure instead of going.

Reply:
@gunnarmorling Sorry, I misunderstood it. Changed it now :smiley: